pandas/io/formats/style.py

```diff
@@ -77,6 +81,8 @@ class Styler:
     See Also
     --------
     DataFrame.style
+        Returns a Styler object containing methods for building
```

Comment:
I don't think this belongs in the `See Also` section - is this addressing some issue?

Reply:
Had this error - Missing description for See Also "DataFrame.style" reference. Hence added this  line of description

Reply:
Oh OK thanks for clarifying. I think this is an issue with the validator as the format *should* match what you see in the docsring guide (specifically small description separated by a colon):

https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-5-see-also

Reply:
Made the changes as requested.